lib/apple/index.js

```diff
@@ -112,8 +116,8 @@ exports.verifyPayment = function (payment, cb) {
 		return cb(null, result);
 	}
 
-
-	verify(apiUrls.production, { json: jsonData }, function (error, resultString) {
+	var url = sandbox ? apiUrls.sandbox : apiUrls.production;
```

Comment:
Sure it works :) I'm just wondering if the added complexity is warranted. Up till now, users never had to provide a boolean to say "sandbox please".